content/en/blog/2020/dns-proxy/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: New Frontiers - Smart DNS Proxying in Istio
```

Comment:
```suggestion
title: Smart DNS Proxying in Istio
```

(Maybe with "new frontiers" in the subtitle?  It feels like a brand where it is, which isn't your intention)

Reply:
aww. I was going for the star trek effect.. venturing into unknown, exploring new frontiers, blah. Where else can we show off this narcism ?